backend/src/main/java/com/bakdata/conquery/models/query/queryplan/ConceptQueryPlan.java

```diff
@@ -98,6 +98,11 @@ protected SinglelineEntityResult result() {
 	@Override
 	public Optional<SinglelineEntityResult> execute(QueryExecutionContext ctx, Entity entity) {
 
+		if (ctx.getQueryDateAggregator().isEmpty()) {
```

Comment:
Das hast du jetzt hier und im ArrayConceptQueryPlan gemacht, sollte das dann nicht iwie besser gekapselt sein, dass es nicht beide machen müssen?

Reply:
Da es in anderen QueryPlänen nicht gemacht werden muss habe müsste ich eine Zwischenklasse erstellen die dann extra dieses handling macht